.github/workflows/test.yml

```
@@ -49,8 +49,35 @@ jobs:
+    - uses: actions/checkout@v7
+    - name: Set up Python 3.12
+      uses: actions/setup-python@v7
+      with:
+        python-version: '3.12'
+    - uses: astral-sh/setup-uv@v7
+      with:
+        enable-cache: true
+        cache-dependency-glob: |
+          **/requirements*.txt
+          **/pyproject.toml
+    - run: uv pip install --system .[test,bohrium] coverage
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2102:info:1:26: Ranges can only match single chars (mentioned due to duplicates)

(shellcheck)